ci-operator/config/openshift/osde2e/openshift-osde2e-main.yaml

```diff
@@ -18,21 +22,34 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: verify
+- as: code-quality-checks
   commands: |
-    set -o pipefail
-
+    export GOFLAGS=""
     export HOME=$(mktemp -d)
     export REPORT_DIR="$ARTIFACT_DIR"
 
-    ci/prow_pr_check.sh
+    make check
+  container:
+    from: src
+- as: verify
+  commands: |
+    export GOFLAGS=""
+    export REPORT_DIR="$ARTIFACT_DIR"
+    export CONFIGS="aws,prod,e2e-suite"
```

Comment:
Is there a reason we need to run the full e2e-suite? Is there a config with a smaller set of core tests we can use? This seems a little excessive for a PR check IMO.

Reply:
We don't have anything smaller but we could make one:
The e2e-suite today:
tests:
  testsToRun:
  - '\[Suite\: e2e\]'
  - '\[Suite\: operators\]'
  - '\[Suite\: service-definition\]'
  - '\[Suite\: app-builds\]'

Reply:
Created a pr-check config in the osde2e repo that only includes the service-definition tests.